### ISSUE TITLE
Cannot install Dracula on Racket 8.1

### ISSUE THREAD

**Issue by zxa4fd** [racket/racket#4129]
**What version of Racket are you using?**
I am using Racket 8.1

**What program did you run?**
I ran this command:
`raco pkg install dracula`
and
`raco setup`

according to [this](https://dracula-lang.github.io/) link

**What should have happened?**
It should have installed the package. Instead
it just says package installation failed.

**Comment by sorawee** [comment]
Can you post the full error message?

**Comment by zxa4fd** [comment]
> Can you post the full error message?

[Here](https://pastebin.com/PJvpWxP7) is the error when I run the first command.

[Here](https://pastebin.com/kH6m8xYE) is the error when I run the second command.

**Comment by jbclements** [comment]
I would transfer this issue to the 
https://github.com/carl-eastlund/dracula
repository.

It looks to me like the pkg-manager fails in trying to compile dracula much earlier. Here's the transcript:
https://pkg-build.racket-lang.org/server/built/fail/dracula.txt

Perhaps you installed this package ... Oh! I see, you tried to install it using Racket 8.1, which was an excellent idea.

I just tried it using v6.3 minimal, which came out at that time, but got this 

```
clements@johnlewis:/tmp/rkttemp/racket/bin$ ./raco pkg install dracula
Resolving "dracula" via http://download.racket-lang.org/releases/6.3/catalog/
ssl-connect: requested protocol not supported;
 SSL not available; check `ssl-load-fail-reason'
  requested: 'auto
  context...:
   /tmp/rkttemp/racket/collects/openssl/mzssl.rkt:611:0: make-raw-context
   /tmp/rkttemp/racket/collects/openssl/mzssl.rkt:482:18
   /tmp/rkttemp/racket/collects/racket/private/more-scheme.rkt:265:2: call-with-exception-handler
   /tmp/rkttemp/racket/collects/openssl/..:261:28
   /tmp/rkttemp/racket/collects/openssl/..:259:25
   /tmp/rkttemp/racket/collects/net/http-client.rkt:224:0
   /tmp/rkttemp/racket/collects/racket/contract/private/arrow-val-first.rkt:324:3
   /tmp/rkttemp/racket/collects/net/url.rkt:77:0: http://getpost-impure-port
   /tmp/rkttemp/racket/collects/net/url.rkt:179:2: redirection-loop
   /tmp/rkttemp/racket/collects/racket/contract/private/arrow-val-first.rkt:324:3
   /tmp/rkttemp/racket/collects/pkg/private/network.rkt:58:3
   /tmp/rkttemp/racket/collects/pkg/private/catalog.rkt:218:0: read-from-server
   /tmp/rkttemp/racket/collects/pkg/private/catalog.rkt:138:9: for-loop
   /tmp/rkttemp/racket/collects/pkg/private/catalog.rkt:135:2: lookup-normally
   /tmp/rkttemp/racket/collects/pkg/private/..:261:28
   /tmp/rkttemp/racket/collects/pkg/private/prefetch.rkt:128:2
   ...
```

In summary, I think it has been a long time since dracula was last used, and my guess is that four to five hours of work from an expert would be required to get it working. That's just a guess, though. Maybe you could write to Carl and he could fix it right up?

**Comment by mfelleisen** [comment]
I pinged Carl on his private email this morning ..

**Comment by zxa4fd** [comment]
Hello, I have posted this [issue](https://github.com/carl-eastlund/dracula/issues/7) in the Dracula repository, but there are no replies. Please advise

**Comment by jbclements** [comment]
Ugh. I wish I had better advice for you, but I would say that at this point, it's a question of finding and/or paying someone to repair the package. I could probably get it running, but it would probably be about 10 hours of work, or about $1500. This is the $64,000 question of open-source development: who's going to maintain it?

I'm going to close this issue, it looks like your freshly created issue will retain a reference to this one, and no information will be lost. Let me know if you think this is a mistake.
